src/bindings/python/src/pyopenvino/graph/model.cpp

```diff
@@ -706,6 +706,19 @@ void regclass_graph_Model(py::module m) {
         return "<" + class_name + ": '" + self.get_friendly_name() + "'\ninputs[\n" + inputs_str + "\n]\noutputs[\n" +
                outputs_str + "\n]>";
     });
+
+    model.def(
+        "__deepcopy__",
+        [](ov::Model& self, py::dict& memo) {
```

Comment:
```suggestion
        [](ov::Model& self, py::dict) {
```
Small nit, there is no need for `memo` in this scenario. It could work without argument.

Reply:
done